### PR TITLE
feat(shows2): supports online shows

### PR DESCRIPTION
**Pull request by dzucconi**
This PR adds support for the "Online Exclusive" shows

### CODE REVIEW

File: src/v2/System/Router/RouterLink.tsx [hunk 1]
```diff
@@ -56,11 +56,28 @@ type RouterLinkMixinProps = BoxProps & {
   textDecoration?: ResponsiveValue<string>
 }
 
+const VALID_FOUND_LINK_PROPS = [
+  "activeClassName",
```
Comment (dzucconi) [inline src/v2/System/Router/RouterLink.tsx:60]: I guess I previously broke active highlighting when I enabled the prop filtering here.

File: src/v2/System/Router/RouterLink.tsx [hunk 2]
```diff
@@ -56,11 +56,28 @@ type RouterLinkMixinProps = BoxProps & {
...
 export const RouterAwareLink: React.FC<LinkPropsSimple & RouterLinkMixinProps> =
   // TODO: Update styled-components types
   // @ts-ignore
   styled(Link).withConfig({
-    shouldForwardProp: (prop, defaultValidatorFn) => defaultValidatorFn(prop),
+    shouldForwardProp: (
+      prop: string,
+      defaultValidatorFn: (prop: string) => boolean
+    ) => {
+      return defaultValidatorFn(prop) || foundValidatorFn(prop)
```
Comment (damassi) [inline src/v2/System/Router/RouterLink.tsx:79]: `FOUND` and `found` feel too close to encoding implementation details. can we just say `routerLinkProps`, et?

**Review by damassi — APPROVED** [review]

File: src/v2/System/Router/RouterLink.tsx (hunk 2 above)
Comment (dzucconi) [reply src/v2/System/Router/RouterLink.tsx:79]: Ah, yeah sure

**Review by damassi — APPROVED** [review]
Lets get this out the door but at some future point be sure to add a few tests (if they're not already there).

**Comment by dzucconi** [comment]
Yeah, there's already specs for City/Index routes. This is just adding a new city, so

**Pull request merged**